### PR TITLE
fix(#2450): highlight for `webp` and `jxl` files

### LINKED ISSUE #2450: `NvimTreeImageFile` highlight not working for `webp` and `jxl` files
### Description

Image files such as `png,jpg` are highlighted with `NvimTreeImageFile` highlight group, but this highlight group doesn't apply for `webp,jxl` image files.

Screenshot_2023-10-07-06-51-26_1920x1080

### Neovim version

```text
NVIM v0.10.0-dev-1278+g1e7e9ee91
Build type: RelWithDebInfo
LuaJIT 2.1.1695653777
```

### Operating system and version

Linux archBTW 6.5.5-arch1-1 #1 SMP PREEMPT_DYNAMIC Sat, 23 Sep 2023 22:55:13 +0000 x86_64 GNU/Linux

### nvim-tree version

master, 85abe29

### Clean room replication

```lua
vim.g.loaded_netrw = 1
vim.g.loaded_netrwPlugin = 1

vim.cmd [[set runtimepath=$VIMRUNTIME]]
vim.cmd [[set packpath=/tmp/nvt-min/site]]
local package_root = "/tmp/nvt-min/site/pack"
local install_path = package_root .. "/packer/start/packer.nvim"
local function load_plugins()
  require("packer").startup {
    {
      "wbthomason/packer.nvim",
      "nvim-tree/nvim-tree.lua",
      "nvim-tree/nvim-web-devicons",
      -- ADD PLUGINS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
    },
    config = {
      package_root = package_root,
      compile_path = install_path .. "/plugin/packer_compiled.lua",
      display = { non_interactive = true },
    },
  }
end
if vim.fn.isdirectory(install_path) == 0 then
  print "Installing nvim-tree and dependencies."
  vim.fn.system { "git", "clone", "--depth=1", "https://github.com/wbthomason/packer.nvim", install_path }
end
load_plugins()
require("packer").sync()
vim.cmd [[autocmd User PackerComplete ++once echo "Ready!" | lua setup()]]
vim.opt.termguicolors = true
vim.opt.cursorline = true

-- MODIFY NVIM-TREE SETTINGS THAT ARE _NECESSARY_ FOR REPRODUCING THE ISSUE
_G.setup = function()
  require("nvim-tree").setup {}
end
```

### Steps to reproduce

1. nvim -nu /tmp/nvt-min.lua
2. Open NvimTree inside any project that has webp files.

### Expected behavior

webp files should have same highlight as png file.

### Actual behavior

webp file has different highlight than png,jpg files.

### PR DESCRIPTION
**Pull request by zootedb0t**
Closes #2450

### CODE REVIEW

File: lua/nvim-tree/renderer/init.lua [hunk 1]
```diff
@@ -51,6 +51,8 @@ local picture_map = {
   jpeg = true,
   png = true,
   gif = true,
+  webp = true,
+  jxl = true,
```
Comment (alex-courtis) [inline lua/nvim-tree/renderer/init.lua:55]: Hrmmm....

I was planning to remove this very-legacy mapping as it should be handled by nvim-web-devicons.

https://github.com/nvim-tree/nvim-web-devicons/pull/318 has been merged. Is that sufficient @zootedb0t ?

Comment (zootedb0t) [reply lua/nvim-tree/renderer/init.lua:55]: Icon for `jxl` files work now, but the highlight still doesn't apply to `jxl and webp` files.

Screenshot_2023-10-08-05-59-25_1920x1080

Comment (alex-courtis) [reply lua/nvim-tree/renderer/init.lua:55]: Ah I see. I forgot about highlight...

**Review by alex-courtis — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**